src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -2117,9 +2136,9 @@ private void RemoveSingleItem(PackageItemRootViewModel vm, DependencyType fileTy
             }
         }
 
-        private bool CanShowAddFileDialogAndAdd()
+        private bool CanAddFiles()
```

Comment:
Why not just remove this redundant property (it's just the inverse of `IsPackageInstalled`) and use `IsPackageInstalled` directly?